sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -615,9 +648,11 @@ public int compare(FileResult first, FileResult second) {
      *
      * <p>Can be called from subclasses that override {@link WriteOperation#finalize}.
      *
-     * <p>Files will be named according to the file naming template. The order of the output files
-     * will be the same as the sorted order of the input filenames.  In other words, if the input
-     * filenames are ["C", "A", "B"], baseOutputFilename is "file", the extension is ".txt", and
+     * <p>Files will be named according to the {@link FilenamePolicy}. The
+     * order of theoutput files
```

Comment:
theoutput; comment misformatted. Also, we don't have a baseOutputFilename anymore, only temp, right?